client/me/purchases/purchases-list-in-dataviews/purchases-data-field.tsx

```diff
@@ -14,6 +14,8 @@ import {
 	BackupPaymentMethodNotice,
 } from '../purchase-item';
 import OwnerInfo from '../purchase-item/owner-info';
+import { MembershipSubscription } from 'calypso/lib/purchases/types';
+import { Icon, MembershipType, MembershipTerms } from '../membership-item';
```

Comment:
I get a compile error here because `MembershipType` and `MembershipTerms` are not exported.